src/Mvc/Mvc.ApiExplorer/src/DefaultApiDescriptionProvider.cs

```diff
@@ -445,6 +446,22 @@ internal static MediaTypeCollection GetDeclaredContentTypes(IReadOnlyList<IApiRe
             return contentTypes;
         }
 
+        internal static IReadOnlyList<string> GetAcceptsContentTypes(IReadOnlyList<IAcceptsMetadata>? requestMetadataAttributes)
```

Comment:
Why is it on the default provider when it's not used by it?

Reply:
It's being used by ``` AddAcceptsRequestFormats``` .  Is this not a good approach as a utility method ?